### PR TITLE
Merging "Feature-master-beta3" features into "Master" branch

### PR DESCRIPTION
**Pull request by maheshkulkarni4**
Created PR to merge the feature-master-beta3 into Master.

### CODE REVIEW

**Review by ghatea — APPROVED** [review]
LGTM.

File: package.json [hunk 1]
```diff
@@ -1,15 +1,15 @@
 {
     "name": "@dell/clarity-react",
-    "version": "0.24.6",
+    "version": "0.25.2",
```
Comment (akate1) [inline package.json:3]: We already have package for "0.25.2" so need to increase version to "0.25.3"

**Review by akate1 — APPROVED** [review]
LGTM

File: package.json (hunk 1 above)
Comment (maheshkulkarni4) [reply package.json]: Done

**Pull request merged**